app/Http/Controllers/EmployeeController.php

```diff
@@ -109,22 +112,19 @@ public function create()
             ->with('branches', Branch::all())
             ->with('departments', Department::all())
             ->with('employment_statuses', $this->employment_statuses)
-            ->with('designations', Designation::all());
+            ->with('designations', Designation::all())
+            ->with('managers', Employee::where('status', '!=', '0')->get())
+            ->with('teams', Team::where('status', '!=', '0')->get());
     }
 
     public function store(Request $request)
     {
         //also do js validation
-        $this->validate($request, [
-            'firstname'      => 'required',
-            'lastname'       => 'required',
+        $request->validate([
             'official_email' => 'required|email|unique:employees',
             'personal_email' => 'required|email|unique:employees',
-            'contact_no'     => 'required|unique:employees|size:11',
-            'gender'         => 'required',
-            'picture'        => 'max:1000',
-
-            // 'cnic' => 'size:13',
+            'contact_no'     => 'required|size:11|unique:employees,contact_no',
+            'picture'        => 'image|mimes:jpg,png,jpeg,gif,svg|max:1000',
         ]);
 
         if (! strstr(strtolower($request->official_email), 'glowlogix.com')) {
```

Comment:
@JoharAlam 
We are making it open source so **"glowlogix.com"** cant be hardcoded, So make it dynamic.

Reply:
@awaidanjum I will fix it